src/eXpansion/Framework/GameShootmania/DataProviders/PlayerExtraDataProvider.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace eXpansion\Framework\GameShootmania\DataProviders;
+
+use eXpansion\Framework\Core\DataProviders\AbstractDataProvider;
+
+/**
+ * Class PlayerDataProvider provides information to plugins about what is going on with players.
+ *
+ * @package eXpansion\Framework\Core\DataProviders
+ */
+class PlayerExtraDataProvider extends AbstractDataProvider
+{
+
+    /**
+     * @param $params
+     */
+    public function onNearMiss($params)
+    {
+        $this->dispatch(__FUNCTION__,
+            [
+                $params['shooter'],
+                $params['victim'],
+                $params['weapon'],
+                $params['distance'],
+                (object)$params['shooterposition'],
```

Comment:
for landmarks you did a nice little model, cand we do the same for position? So we have autocomplete

Reply:
ok